packages/spotlight/Spottable/Spottable.js

```diff
@@ -288,9 +254,14 @@ const Spottable = hoc(defaultConfig, (config, Wrapped) => {
 			return notPrevented && allow;
 		}
 
+		isActionable = () => {
+			return !this.props.disabled && !this.props.spotlightDisabled;
+		}
+
 		handle = handle.bind(this)
 
 		handleKeyDown = this.handle(
+			this.isActionable,
```

Comment:
Should we still allow the spotlight directional events (e.g. `onSpotlightLeft`) when focused but disabled? Seems feasible that consumers would want the same support for managing 5-way navigation from a component regardless of its `disabled` state.